insights/parsers/yumlog.py

```diff
@@ -28,124 +28,21 @@
     ``python -m insights.parsers.yumlog``
 
 Examples:
-    >>> content = '''
-    ... May 23 18:06:24 Installed: wget-1.14-10.el7_0.1.x86_64
-    ... Jan 24 00:24:00 Updated: glibc-2.12-1.149.el6_6.4.x86_64
-    ... Jan 24 00:24:09 Updated: glibc-devel-2.12-1.149.el6_6.4.x86_64
-    ... Jan 24 00:24:10 Updated: nss-softokn-3.14.3-19.el6_6.x86_64
-    ... Jan 24 18:10:05 Updated: 1:openssl-libs-1.0.1e-51.el7_2.5.x86_64
-    ... Jan 24 00:24:11 Updated: glibc-2.12-1.149.el6_6.4.i686
-    ... May 23 16:09:09 Erased: redhat-access-insights-batch
-    ... Jan 24 00:24:11 Updated: glibc-devel-2.12-1.149.el6_6.4.i686
-    ... '''.strip()
-    >>> from insights.tests import context_wrap
-    >>> yl = YumLog(context_wrap(content))
-    >>> e = yl.present_packages.get('nss-softokn')
-    >>> e.pkg.release
-    '19.el6_6'
-    >>> e = yl.present_packages.get('openssl-libs')
-    >>> e.pkg.name
-    'openssl-libs'
-    >>> e.pkg.version
-    '1.0.1e'
-    >>> len(yl)
-    8
-    >>> indices = [e.idx for e in yl]
-    >>> indices == range(len(yl))
-    True
+     >>> YumLog.filters.append('wrapper')
+     >>> YumLog.token_scan('daemon_start', 'Wrapper Started as Daemon')
+     >>> logs = shared[YumLog]
```

Comment:
Since `shared` is deprecated in core v3.0, would you please to remove this line?
And better add a test method to test the examples in docstring. 

See: https://github.com/RedHatInsights/insights-core/blob/869bbb4c745fd6559a7276dc88a4400a55fbcde3/insights/parsers/tests/test_hostname.py#L68

Reply:
Removed.